static/js/util.js

```diff
@@ -230,6 +230,10 @@ exports.is_mobile = function () {
 exports.prefix_sort = function (query, objs, get_item) {
```

Comment:
IMHO creating a new function and adding emoji related sorting logic there would be a cleaner approach, what do you think?

Reply:
I did in my [first commit of this PR](https://github.com/zulip/zulip/pull/7680/commits/4344caac218215d9fdd2b0f920c54e8fe699cb89) but @rishig felt that using this function itself would be a better approach.

Reply:
I looked at Rishi's comment, he actually meant that in the new function you should first call the original prefix sort function and then manipulate its output to give preference to popular emojis. You were actually repeating the function's logic which is to be avoided.

Reply:
@HarshitOnGitHub Thanks for that! I've pushed a few changes in which a separate function uses the prefix sort and modifies this result for displaying popular emoji first. :)